CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ We use [semantic versioning](http://semver.org/):
 # 2.5.0
 
 - [feature] new `--timeout` command line argument for specifying request timeouts in seconds
+- [feature] support for faster coverage extraction from XCResult bundles with [XCode 13.3 command line tools](https://developer.apple.com/documentation/xcode-release-notes/xcode-13_3-release-notes) (see 82004604)
+- [feature] release for MacOS
```

Comment:
Since we don't have build machines running macOS we'll have to build it locally after the merge and then attach a Zip for macOS to the release page. I can take care of this.